### DISCUSSION TITLE
Isn't the session timeout too short?

### DISCUSSION THREAD

**Discussion by Wintus** [TechnitiumSoftware/DnsServer#357] [Ideas]
I feel that the web app's session is way too short.

I found a code seems to control the duration:
https://github.com/TechnitiumSoftware/DnsServer/blob/e3190db4d16e36f4f03525fad56c83edd4a7e5c0/DnsServerCore/UserSession.cs#L28

How about lengthen it or make it configurable?

**Comment by ShreyasZare** [comment]
Thanks for the post. 

There is a trick to keep your session alive forever by keeping the Dashboard > Last Hour view open. When in this Last Hour view, a javascript timer keeps refreshing the data every 60 sec keeping the session alive.

I will plan to add an option for configuring this in later release.
